src/huggingface_hub/commands/lfs.py

```diff
@@ -126,6 +133,8 @@ def run(self):
             write_msg({"error": {"code": 32, "message": "Wrong lfs init operation"}})
             sys.exit(1)
 
+        ref_concurrency = max(init_msg.get("concurrenttransfers", 4), 1)
```

Comment:
I'm not sure of the behavior of git lfs when setting `lfs.customtransfer.multipart.concurrent` to `false`, I'd rather not rely on what it sends us and set a value we know would works best in most cases.

```suggestion
```

Reply:
[git lfs spec](https://github.com/git-lfs/git-lfs/blob/main/docs/custom-transfers.md#defining-a-custom-transfer-type) says,

> lfs.customtransfer.\<name\>.concurrent
If true (the default), git-lfs will invoke the custom transfer process multiple times in parallel, according to lfs.concurrenttransfers, splitting the transfer workload between the processes.
If you would prefer that only one instance of the transfer process is invoked, maybe because you want to do your own parallelism internally (e.g. slicing files into parts), set this to false.

That means, if we have 3 three object files (not chunks or parts), and lfs.concurrenttransfers defaults to 8, git-lfs will  send them in 3 worker processes, with the rest of 5 idle processes.

As we do parallelism internally in one process, so set it to `false`. But we can use value of `lfs.concurrenttransfers` to control the concurrency for different environment, instead of hard coding for all.

Reply:
> As we do parallelism internally in one process, so set it to false.

Yes this a good idea, not questioning that 😄 

I would avoid using `lfs.concurrenttransfers` as we don't know who will set it (user or the install script or maybe some other config) and is related to git-lfs. Especially so, since you disable lfs's concurrency, it logically doesn't make sense to use a configuration parameter associated to git lfs's concurrency.

We can always have an environment variable if we really want to let the user override it, I feel it's best if we set a default value we know works well.